src/main/scala/main/Bot.scala

```diff
@@ -172,13 +188,8 @@ object Bot {
     }
   }
 
-  def callCommand(l: List[String]): String = l.head match {
-    case "!prs" => parseTeam(l.tail)
+  def callCommand(l: List[String], label: String): String = l.head match {
+    case "!prs" => getPullRequests(label)
```

Comment:
Can we keep the option to pull PRs for other channels by checking the for other arguments and then ignore the current channel?